src/systemd.h

```diff
@@ -0,0 +1,32 @@
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <sys/xattr.h>
+#include <unistd.h>
+#include <systemd/sd-bus.h>
+
+#ifndef __LIBC_SYSTEMD
+#define __LIBC_SYSTEMD
+
+/**
+ * Function to create a scope of specified name within the specified slice; the scope is delegated if

```

Comment:
Look at my comment in the makefile patch I shared with you.  Libcgroup uses doxygen-style comments in the *.h files.
https://github.com/drakenclimber/libcgroup/commit/98e8f16230048831e7ba30f0092ff24148a3076b#diff-874ad12789566e63ecf1207b33dcf475df61d3036be3ef7fc9f91cac883399b2R15

Reply:
Fixed; header and footer also copied over so the naming of the ifndef and comments surrounding those should now be correct.